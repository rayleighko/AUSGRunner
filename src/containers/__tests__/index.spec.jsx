import React from 'react'
import {shallow} from 'enzyme'
import {TRexContainer} from "../"

describe('Containers', () => {
	describe('T-Rex Container', () => {
		it('should render', () => {
			const tree = shallow(<TRexContainer/>)
			expect(tree).toMatchSnapshot()
		})
	})
})