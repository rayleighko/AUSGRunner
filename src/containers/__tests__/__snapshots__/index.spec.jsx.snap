// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Containers CreatedBy Container should render 1`] = `ShallowWrapper {}`;

exports[`Containers Home Container should render 1`] = `ShallowWrapper {}`;

exports[`Containers PlayGround Container should render 1`] = `ShallowWrapper {}`;
